encoders/image/CLIPImageEncoder/tests/__init__.py

```diff
@@ -0,0 +1 @@
+#
```

Comment:
remove this, it can be empty with no problem

Reply:
I still see this file non-empty

Reply:
This file should be empty

Reply:
yes, sorry